packages/epilot-sdk/src/email-setting-client.ts

```diff
@@ -0,0 +1,6 @@
+import { getClient } from '@epilot/email-settings-client';
```

Comment:
Module name should match the  package name

`packages/epilot-sdk/src/email-setting-client.ts`